source/elements/oneDAL/source/glossary.rst

```diff
@@ -0,0 +1,277 @@
+        ordering between values.
+
+        **Example:** student's grade
+
+    Ratio feature
+        A :term:`continuous feature` with defined operations of equality,
+        comparison, addition, substraction, multiplication and division.
+        Has a zero value element which means the absence of value.
+
+        **Example:** the height of a tower
+
+    Regression
+        A :term:`supervised machine-learning problem<Supervised learning>` of
+        assigning :term:`continuous<Continuous feature>`
+        :term:`responses<Response>` to :term:`feature vectors<Feature vector>`.
+
+        **Example:** prediction of the temperature based on weather conditions
+
+    Response
+        A property of some real object or event which value is target to be
+        predicted in the :term:`supervised learning` task. While the
+        :term:`feature` is an input in the machine learning task, the response
+        is one of the outputs can be made by the :term:`model` on the
+        :term:`inference` stage.
+
+        **Synonym:** dependent variable
+
+    Supervised learning
+        :term:`Training` process with a :term:`dataset` containing information
+        on dependencies between :term:`features<Feature>` and
+        :term:`responses<Response>`. The goal is to get a :term:`model` of
+        dependencies between input :term:`feature vector` and
+        :term:`responses<Response>`.
+
+    Training
+        A process of creating a :term:`model` based on information extracted
+        from a :term:`training set`. Resulting :term:`model` is selected in
+        accordance with some quality criteria.
+
+    Training set
+        A :term:`dataset` used at the :term:`training` stage to create a
+        :term:`model`.
+
+    Unsupervised learning
+        :term:`Training` process where :term:`training set` does not contain
+        information on dependencies between :term:`feature vectors<Feature
+        vector>` and :term:`responses<Response>`. The goal is to get a
+        :term:`model` matching :term:`feature vectors<Feature vector>` into
+        groups.
+
+|dal_short_name| terms
+======================
+
+.. glossary::
+    :sorted:
+
+    Accessor
+        An |dal_short_name| concept of an object, which provides an access in the
+        special :term:`data format` to the data of another object.
+
+    Batch Mode
+        The computation mode for an algorithm in |dal_short_name|, where all the
+        data needed for computation is available at the start and fits the
+        memory of the device on which the computations are performed.
+
+    Builder
+        An |dal_short_name| concept of an object which encapsulates the creation
+        process of another object and provides an ability to create it
+        iteratively.
+
+    Contiguous data
+        A special property of the :term:`data format`: the data are stored as
+        one contiguous memory block.
```

Comment:
Just because it is a single attribute of data format. It does not fully describe data format: data can be continuous, heterogeneous, stored in row-major layout with feature types (double, float, int). That's the full description of data format.

Reply:
would this be correct?

```
Data that is stored as one contiguous memory block.
One of the characteristics of a :term:`data format`.
```

Reply:
Yes, but I'd like to write "data that are stored..." because data is a plural noun. Correct me please, if I wrong.

Reply:
there's a controversy :) 
generally both `data are`/`data is` are ok as long as you use them consistently, but there's a tendency towards data as a singular mass noun (similar to information)
if you look at other places in this doc, you'll see data used as a singular